Sources/MongoSwiftSync/MongoCollection.swift

```diff
@@ -63,4 +63,41 @@ public struct MongoCollection<T: Codable> {
     public func drop(options: DropCollectionOptions? = nil, session: ClientSession? = nil) throws {
         try self.asyncColl.drop(options: options, session: session?.asyncSession).wait()
     }
+
+    /**
+     * Renames this collection with the specified options on the server. This method will return a handle to the
```

Comment:
The "with the specified options" sort of sounds like it will be used as a filter to decide which collection to rename. I think we can just drop that part from the docstring and it should still be clear enough. The listing of the parameters below should be enough to indicate the command is configurable.